Libraries/Components/Button.windows.js

```diff
@@ -116,6 +147,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    let content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      const tabIndex = this.props.tabIndex || 0;
+      const windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref = {(ref) => this._setFocusableRef(ref)}
```

Comment:
>ref = {(ref) => this._setFocusableRef(ref)} [](start = 10, length = 43)

Avoid inline functions in JSX when possible. To read about why it's bad for refs in particular, read here: [https://reactjs.org/docs/refs-and-the-dom.html#caveats](https://reactjs.org/docs/refs-and-the-dom.html#caveats)